### PR TITLE
#692 Documented that commands are optional in .rultor.yml

### PR DESCRIPTION
**Pull request by alevohin**
#692 Documented that commands are optional in .rultor.yml

### CODE REVIEW

**Comment by alevohin** [comment]
@alex-palevsky CI build failed due to #733 (as minimum for travis-ci and semaphoreci). I think there are 2 ways:
1) Wait for #733
2) Find someone for CR (the only stable build is under  continuous-integration/wercker)
What do you think?

**Comment by alex-palevsky** [comment]
Many thanks, I'll find someone to review it before we merge

**Comment by alex-palevsky** [comment]
@longtimeago review this plz

File: src/jekyll/_posts/2014/jul/2014-07-13-reference.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ description:
 Rultor is configured solely through the YAML `.rultor.yml` file stored in the
 root directory of your Github repository. There is no control or management
 panel. Everything you want to say to Rultor is placed into your `.rultor.yml`
-file.
+file. This file may be empty but it must exist.
```
Comment (longtimeago) [inline src/jekyll/_posts/2014/jul/2014-07-13-reference.md]: Coma missed.
```
This file may be empty, but it must exist.
```

File: src/jekyll/_posts/2014/jul/2014-07-13-reference.md [hunk 2]
```diff
@@ -239,6 +239,12 @@ The list of Github accounts able to give commands to Rultor is specified in
 commands. Configured commanders don't replace collaborators. In other words,
 Github collaborators *and* accounts mentioned here are allowed to give commands.
 
+Any of these commands are optional in `.rultor.yml`. You can invoke these commands
```
Comment (longtimeago) [inline src/jekyll/_posts/2014/jul/2014-07-13-reference.md]: Better to say ```All these commands are optional in ...```

Comment (longtimeago) [inline src/jekyll/_posts/2014/jul/2014-07-13-reference.md]: no need in this change

**Comment by alevohin** [comment]
@longtimeago thanks! Please review one more time.

**Comment by longtimeago** [comment]
@alevohin thanks

**Comment by longtimeago** [comment]
@rultor please, merge

**Comment by yegor256** [comment]
@rultor merge it

**Comment by alex-palevsky** [comment]
@longtimeago I just added **18 mins** to your account, many thanks for your contribution..; added +18 to your rating, now it is equal to [+1249](http://www.netbout.com/b/35024?open=rating)

**Comment by alex-palevsky** [comment]
@rultor deploy now pls

**Pull request merged**